go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/splunk/splunk-operator
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.17.7
+	github.com/aws/aws-sdk-go v1.30.12
+	github.com/coreos/prometheus-operator v0.38.3 // indirect
```

Comment:
Question similar to aws-sdk-go [discussion](https://github.com/splunk/splunk-operator/pull/267#discussion_r585112183) .. does updating to latest version  of prometheus operator cause build failures or other problems?

Reply:
Yes, upgrading to the latest version of prometheus operator causes dependency issues and build issues. v0.38.3 is the latest version that will work without further changes such as upgrading the operator sdk version.